### ISSUE TITLE
1.20.4 Cannot launch.

### ISSUE THREAD

**Issue by Duxtorm** [Overcontrol1/RandomFishing#7]
When I try to launch the 1.20.4 version of the mod, I get an error message saying it requires **0.7.9+1.20.4 of Polymer** as well as **2.2.0+1.20.3-rc1 of Server-translations-api**

Now, it mentioned Polymer even though I already have Polymer 0.7.9, I tried both Curse Forge and Prism, but same issue.
I also have no idea what Server-translations-api is or how to find it, nor was there any mention of it being a dependency on Modrinth.

Closest thing I could find was this
https://www.curseforge.com/minecraft/mc-mods/server-translation-api
But it's only for 1.19.4 latest..

**Comment by tessitalisman** [comment]
Here's the github. 

https://github.com/NucleoidMC/Server-Translations/releases/download/v2.2.0%2B1.20.3-rc1/Server-Translations-2.2.0+1.20.3-rc1.jar

**Comment by tessitalisman** [comment]
Nevermind, seems to be a problem with random fishing itself not seeing the mods. At least I assume

**Comment by Overcontrol1** [comment]
Yeah, I perhaps did an oopsie. Should be fixed in 1.3.1 and above.
